### PR TITLE
Adds a bomb collar implant

### PR DESCRIPTION
**Pull request by Time-Green**
https://forums.yogstation.net/index.php?threads/bomb-collars-need-a-massive-buff.11506/#post-111705

Adds an implant that has the status feature from the detonator, as well as an extre feature wich allows it to bind people to you. If you die, they die with you.

:cl:
rscadd: Adds a bomb collar implant that has the status feature from the detonator, and can bind people to you, so that if you die, they die with you.
add: The above is added to the security locker. weeeeee changelog.
/:cl:

### CODE REVIEW

**Comment by ShadowDeath6** [comment]
making it a remote implant is stupid, just have it detonate people upon the implanted person's death, we have a remote for a reason, and that might make people not kill the HoS if they couldn't blow anyone up while alive and without the remote

**Comment by Time-Green** [comment]
That's the thing, if the HoS for say has that implant and you're bomb collared, you're gonna wanna not attack them.
When you got bombcollared, you lost your round, go do your job, you got a second chance

EDIT: wait, what do you mean? You can still use the remote, you need to link them to the remote, and then use the remote to update the implant with the latest collars.

**Comment by ShadowDeath6** [comment]
how about have the remote let you do shit to people while you're alive and the implant do shit when you die instead of making the implant literally better in every single respect because it's a hands free remote

**Comment by Time-Green** [comment]
I guess so. It's an easy change.

File: code/game/objects/items/weapons/implants/implantcollar.dm [hunk 1]
```diff
@@ -0,0 +1,114 @@
+/obj/item/weapon/implant/bombcollar
+	name = "collar console implant"
+	desc = "You die with me."
+	origin_tech = "materials=4;magnets=4;programming=4;biotech=4;syndicate=5;bluespace=5"
```
Comment (ShadowDeath6) [inline code/game/objects/items/weapons/implants/implantcollar.dm]: mate is this the same tech levels as a powersink and a bluespace crystal at roundstart

**Comment by Zostroll** [comment]
With syndicate level five all you are gonna get is science annoying hos till they get it to decon.

File: code/game/objects/items/weapons/implants/implantcollar.dm (hunk 1 above)
Comment (Time-Green) [reply code/game/objects/items/weapons/implants/implantcollar.dm]: My mistake, I forgot to change that.

**Comment by Time-Green** [comment]
Fixed the origin tech and the implant now only has Status and Bind.

**Pull request merged**